### ISSUE TITLE
complain error

### ISSUE THREAD

**Issue by jpfree** [rkubera/ESP8266-RC-Transmitter-Receiver#3]
i use the nodemcu v3.0,when i compalin this scrip in  Arduino IDE, it have a error:
error: 'rx_setup' was not declared in this scope; did you mean 'setup'?
what can i do ,thanks!
![essr](https://user-images.githubusercontent.com/10045967/125026105-7d8d1600-e0b6-11eb-9c06-eda8c0f5605a.png)

Arduino:1.8.13 (Windows 10), 开发板："Generic ESP8266 Module, 80 MHz, Flash, Disabled (new aborts on oom), Disabled, All SSL ciphers (most compatible), 32KB cache + 32KB IRAM (balanced), Use pgm_read macros for IRAM/PROGMEM, dtr (aka nodemcu), 26 MHz, 40MHz, DOUT (compatible), 1MB (FS:64KB OTA:~470KB), 2, nonos-sdk 2.2.1+100 (190703), v2 Lower Memory, Disabled, None, Only Sketch, 115200"

D:\esp6288\esp8266-wifi-car\ESP8266-RC-Transmitter-Receiver\ESP8266_tx_rx\ESP8266_tx_rx.ino: In function 'void setup()':

ESP8266_tx_rx:154:3: error: 'rx_setup' was not declared in this scope; did you mean 'setup'?

  154 |   rx_setup();

      |   ^~~~~~~~

      |   setup

D:\esp6288\esp8266-wifi-car\ESP8266-RC-Transmitter-Receiver\ESP8266_tx_rx\ESP8266_tx_rx.ino: In function 'void loop()':

ESP8266_tx_rx:164:3: error: 'rx_loop' was not declared in this scope; did you mean 'rx_ip'?

  164 |   rx_loop();

      |   ^~~~~~~

      |   rx_ip

exit status 1

'rx_setup' was not declared in this scope; did you mean 'setup'?

**Comment by rkubera** [comment]
Did you download all sources from repo? In Arduino IDE you have opened only 1 file - ESP8266_tx_tx.ino. Function rx_setup is implemented in RX.ino file. Download full source code, reopen project ant try again, plesae.

Regards, Radek.

**Comment by jpfree** [comment]
yes ,i download all sources from repo.but when i open ESP8266_tx_rx.ino ,the arduino ide say must move to the file folders named ESP8266_tx_rx..
![QQ截图20210713164659](https://user-images.githubusercontent.com/10045967/125421566-b2bca3b6-444e-45ba-95f0-bffc2f11bdb0.png)

thanks for you answer.

**Comment by rkubera** [comment]
But check, if all files are in destination folder (after move) ,please.
In my opinion only ESP8266_tx_rx.ino file was moved.
Regards, Radek.
